snap/local/exporter-wrapper.bash

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eu
+
+PROPERTIES=$(cat "${SNAP_COMMON}"/exporter.properties)
+BOOTSTRAP_SERVERS=$(echo "${PROPERTIES}" | grep bootstrap.servers | sed 's/bootstrap\.servers=//g')
+SECURITY_PROTOCOL=$(echo "${PROPERTIES}" | grep security.protocol | sed 's/security\.protocol=//g')
+SASL_JAAS_CONFIG=$(echo $PROPERTIES | grep sasl.jaas.config | sed 's/sasl\.jaas\.config=//g')
+PASSWORD=$(echo $SASL_JAAS_CONFIG | grep -oP "password\=\"([a-zA-Z0-9]+)" | cut -d "\"" -f 2)
```

Comment:
**thought** Should we error out if any of this variable are not defined? I believe we should at least notify if the exporter.properties file does not exist, as I suppose this would totally break everything else